app/models/partners/served_area.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: served_areas
+#
+#  id                 :bigint           not null, primary key
+#  client_share       :integer
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  county_id          :bigint           not null
+#  partner_profile_id :bigint           not null
+#
+module Partners
+  class ServedArea < ApplicationRecord
```

Comment:
It's a bit confusing to have something under the `Partners::` module whose table doesn't start with `partner_`. Can we pick one or the other?

Reply:
prefixed partner_ to the table name